### ISSUE TITLE
Proper classification

### ISSUE THREAD

**Issue by JacksonCampbell99** [codyogden/killedbygoogle#1167]
For example, Google+ was killed for personal use but became Currents, an enterprise application. Those aren't two separate services. Your kill description says that Currents was similar to Google+, when in fact it was Google+ rebranded (https://9to5google.com/2019/04/10/google-currents-rebrand/). There also is no note on the Google+ kill description stating the Enterprise rebrand.
Google Play Movies and TV is listed as dead when in fact it was rebranded back to Google TV. It is the same application with an update for the new logo and name. Your description doesn't reflect that at all. The Wikipedia page reflects it accurately.
<img width="511" alt="image" src="https://user-images.githubusercontent.com/76508020/155217834-4eaec886-049d-4167-ade0-98b84a5637fb.png">
https://www.theverge.com/2020/9/30/21492549/google-tv-app-play-movies-name-change-chromecast
Your description would be correct if it was listed as Google Play Movies & TV for smart TVs, which was replaced by YouTube on smart TVs, but since that is not listed, then your description is false since the mobile app was not replaced by YouTube but merely rebranded along with the Chromecast rebrand/new Google TV and updated with some new features including a remote for the physical Google TV.
https://arstechnica.com/gadgets/2021/04/google-is-killing-google-play-movies-tv-on-smart-tvs/

Those were a couple examples. Rebrands like Google Play Movies & TV to Google TV should not be on there. Ones like Google+ that lose part of their purpose or that merge with other services would make sense to be listed but should be designated properly. For example, Chat is a rebrand of Hangouts (which still lives for now) built on a new application but with the same chat history and data, so when Hangouts dies, it technically still lives on with a different UI.

**Comment by codyogden** [comment]
Thank you for the issue. I'm going to choose to ignore the tone in the issue, but I will state that there are better ways to communicate one's opinions that do not involve using an accusatory "you."

I've fielded the topic of "rebrands" many times, and it's my opinion that none of these qualify. I agree that true rebrands do not belong in Killed by Google. See #204. Things we should ask about rebrands are, "Is the product exactly the same and simply consist of a new name, or has the product's features been altered or had its availability rescinded during this process?" From my understanding, the examples above either lost features or availability.

Google Play Movies & TV did not just disappear from "smart TVs," it was pulled from multiple platforms. That is a _significant_ altering of a product's availability. If it was simply a "rebrand" those platforms would have continued with the same application with the new name.

Google+/Google Currents provided similar social networking experiences, but Currents lacked the ability to interact with anyone outside of a user's organization. That is a _significant_ altering of the product's features and not simply a "rebrand."

Chat/Hangouts--by your own admission, I might add--are not the same product. Migrating data from Hangouts into Chat only reduces the impact of the first product shutting down. Hangouts is dead and Chat--a brand new product--replaced it. It was not rebranded.

As I mentioned above, #204 is a great example of what I consider a rebrand. The product, AdWords, became Google Ads. It was materially--in both features and availability--the exact same product with a new name and branding.

All that said, I empathize that using a single sentence is difficult to squeeze in the entire context of a product's life. It doesn't provide everyone with additional context about the history of that product. But there is good news. A new version of Killed by Google is actively in the works that will have expanded content beyond a single sentence description. We'll be able to list related products and be able to note predecessors and successors.

Have a wonderful weekend, Jackson!

**Comment by JacksonCampbell99** [comment]
Thanks for the reply. Sorry about the tone of the earlier post. While I understand where you come from, I still don't necessarily agree. I come from the perspective of striving for accuracy and truth.

I must be missing something on Google Play Movies & TV. From the app perspective, its section lives on in the Play Store and the Android app continues on under a simplified name that came through a regular app update. The application and version numbers continue on. On iOS the application lives on under the same [name](https://apps.apple.com/us/app/google-play-movies-tv/id746894884) as well. So, I don't know what you meant about it not just disappearing from smart TVs. While the TV app is gone, it is now integrated into some TVs the same way Alexa or Google Assistant is. Maybe I would agree that it is more dead if I knew where else it was killed, but still being on Android and iOS, it seems like that is the main place it was at anyway.

While Google+ may be "dead" enough to be on the site for the reasons you stated, it as well is the same application with a rebrand that came through an update. It is still installed on my devices and in the exact same place on home screens as it was when I used it with my personal accounts. The limited ability to not use it on personal accounts does warrant it a place on your website, but that doesn't change that it is the same application. I think you know that, I'm just clarifying.

Hangouts and Chat are slightly different. Hangouts still lives currently, but we can assume it will end at some point. I stated that they were different applications (I didn't say different products), which they are. But Chat is simply a rebrand with a revamp of Hangouts. It is not new data, new conversations. I reply to a chat one minute on Hangouts and the next minute on Chat, while the people I am chatting with may be using one or both. Chat and Hangouts are different UIs to interact with the same data. What we can agree on is that Hangouts as the medium to access the data will die. A similar example would be that Instagram DMing has never died but multiple applications to access it have. The DMs themselves never died and the servers didn't change.
